test/dvfinterface.test.js

```diff
@@ -15,13 +17,17 @@ const quantum = 10 ** 8
 
 
 contract('DVFInterface', function (accounts) {
-  let interface, mockStark, token
+  let interface, interfaceWithPermit, mockStark, mockStarkWithPermit, token, tokenWithPermit
 
   beforeEach('redeploy contract', async function () {
     token = await MintableERC20.new('TestToken', 'TEST')
+    tokenWithPermit = await PermitMintableERC20.new('TestTokenWithPermit', 'TESTPERMIT')
     mockStark = await MockStarkExV2.new(token.address)
+    mockStarkWithPermit = await MockStarkExV2.new(tokenWithPermit.address)
     interface = await deployProxy(DVFInterface, [mockStark.address])
+    interfaceWithPermit = await deployProxy(DVFInterface, [mockStarkWithPermit.address])
```

Comment:
there extra contract creations will be redundant if permit is added to the existing mock token